endpoint/14-draft.json

```diff
@@ -219,6 +219,10 @@
               "mastodon": {
                 "description": "Mastodon username. Example: @ordnung@chaos.social",
                 "type": "string"
+              },
+              "matrix": {
+                "description": "Matrix username. Example: <samp>@matrix-bot:matrix.org</samp>",
```

Comment:
[RFC2606](https://tools.ietf.org/html/rfc2606) lists reserved domain names to be used in documentation, for example `example.org`, I would use such a domain name in this document in order to prevent someone accidentally using them.

Reply:
Thanks, changed.